### PR TITLE
fix: fix issue excel_export_template i18n

### PR DESCRIPTION
**Pull request by Malyue**
#### What this PR does / why we need it:
fix issue excel_export_template i18n

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=549175&iterationID=12783&type=BUG)

#### Specified Reviewers:

/assign @sfwn @iutx 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | fix issue excel_export_template i18n             |
| 🇨🇳 中文    |   修复下载导入模版文件中的字段国际化处理           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func GenerateSampleIssueSheetModels(data *vars.DataForFulfill) []vars.IssueSheet
 	requirementCommon := common
 	requirementCommon.ID = 1
 	requirementCommon.IssueType = pb.IssueTypeEnum_REQUIREMENT
-	requirementCommon.Content = "这个一个需求，包含一个任务"
+	requirementCommon.Content = data.I18n(fieldRequirementContent)
 	requirementCommon.State = "进行中"
```
Comment (iutx) [inline internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go:61]: Field `State` is this also a problem?

File: cmd/erda-server/conf/i18n/issue-excel.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ zh:
   # Common
   Common: 通用字段
   ID: ID
-  IterationName: 迭代
+  s: 迭代
```
Comment (iutx) [inline cmd/erda-server/conf/i18n/issue-excel.yaml:5]: The reference `s` is not very clear.

**Review by iutx — COMMENTED** [review]
@Malyue

File: cmd/erda-server/conf/i18n/issue-excel.yaml (hunk 2 above)
Comment (Malyue) [reply cmd/erda-server/conf/i18n/issue-excel.yaml]: This is a careless operation and has been fixed

File: internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go (hunk 1 above)
Comment (Malyue) [reply internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go:61]: The field `State` has not translated in other function，so i delete the translation for `State`

Comment (Malyue) [reply internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go:61]: image

File: cmd/erda-server/conf/i18n/issue-excel.yaml [hunk 3]
```diff
@@ -71,6 +71,15 @@ zh:
   invalid: 无效的
   ImportFailTip: 请下载文件，检查错误信息(最后一列)
 
+  # template
+  RequirementContent: 这是一个需求，包含一个任务
```
Comment (sfwn) [inline cmd/erda-server/conf/i18n/issue-excel.yaml:75]: add prefix: template.xxx

File: internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/field.go [hunk 4]
```diff
@@ -59,6 +59,14 @@ const (
 
 	// Error
 	fieldError = "Error"
+
+	// Template
+	fieldRequirementContent = "RequirementContent"
```
Comment (sfwn) [inline internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/field.go:64]: Move into another const group.
`templateKeyRequirementContent`

File: internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go [hunk 5]
```diff
@@ -34,10 +34,10 @@ func GenerateSampleIssueSheetModels(data *vars.DataForFulfill) []vars.IssueSheet
 
 	common := vars.IssueSheetModelCommon{
 		ID:                 0,
-		IterationName:      data.IterationMapByID[-1].Title,
+		IterationName:      data.I18n(fieldToBePlaned),
```
Comment (sfwn) [inline internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go:37]: set iteration title when initializing.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**